heal.js

```diff
@@ -25,11 +25,21 @@ var ClusterManager = require('./lib/cluster.js');
 var assertNoError = require('./lib/util.js').assertNoError;
 var program = require('commander');
 
+function safeParseInt(number, defaultValue) {
+    number = parseInt(number, 10);
+    if (isNaN(number)) {
+        return defaultValue;
+    } else {
+        return number;
+    }
+}
+
 function main() {
     program
         .description('Start a partition heal coordinated by the target node')
         .option('--tchannel-v1')
-        .option('--verbose')
+        .option('--quiet')
+        .option('--tries <tries>', 'Number of times to try the heal', safeParseInt, 10)
```

Comment:
the default that is currently set is `10`. The last argument of option is the default initial value. This value is also passed into `safeParseInt` as the second argument which I use when the number passed in cannot be parsed as a number.